grr/server/grr_response_server/gui/ui/components/flow_args_form/osquery_query_helper/osquery_table_specs.ts

```diff
@@ -0,0 +1,52 @@
+import tablesJson from '../../../../static/osquery_table_specs_4_5_1.json';
```

Comment:
Unfortunately this won't work with Bazel/Closure compiler. Can we wrap the data to make it a legitimate TS file?